web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -140,7 +146,57 @@ export default defineComponent({
       }
     });
 
-    onMounted(() => {
+    const page = ref(Number(ctx.root.$route.query.pos) || 1);
+    const pages = ref(1);
+    const nextDandiset : Ref<any[]> = ref([]);
+
+    const djangoDandisetRequest: Ref<Paginated<Dandiset> | null> = ref(null);
```

Comment:
I think we can remove this, since it's no longer used anywhere outside of `fetchNextPage`
```suggestion
```